### DISCUSSION TITLE
How to get value from different rows?

### DISCUSSION THREAD

**Discussion by zm14** [Budibase/budibase#6566] [Help]
When creating Formula column with JavaScript, with $("column_name") you get the value of different column in the same row, how to get the value from different row?

**Comment by melohagan** [comment]
Hi @zm14 

This would not possible with Formulas, as they apply on a per row basis.

With that said, if you are grabbing the column from a related table (https://docs.budibase.com/docs/formula#fields-from-relationships)
Then you could adjust the index, or do a filter to grab a specific row on a condition - but again this is for relational data, not data within the same table.
There is a feature request here to allow internal relationships: https://github.com/Budibase/budibase/issues/5635

**Reply by zm14** [reply]
Yes I see its not possible with formulas.
When grabbing a column from another table, how to access the other rows in that table?  
I just want to check if the value that is entered in a field already exist in some other row, can this be done with Actions maybe?

**Comment by melohagan** [comment]
Hi @zm14 

Depending on what you're looking to do, you may be able to use the **Is in** filter. This was recently added in the latest Budibase release!

![isinfilter](https://user-images.githubusercontent.com/101575380/179220306-605333c6-1577-4a29-8919-2d1d5ebe988c.gif)

In this example, I'm using the Conditional UI (https://docs.budibase.com/docs/conditions) 

<img width="209" alt="Screenshot 2022-07-15 at 13 12 33" src="https://user-images.githubusercontent.com/101575380/179220556-d89dd44b-d254-444c-8969-1c6d5c6df089.png">

**Condition for the paragraph**
![Screenshot 2022-07-15 at 13 13 08](https://user-images.githubusercontent.com/101575380/179220634-46de7d3c-cb90-4336-b666-5978f8fb7973.png)

`Show Component IF 'TextField' IS IN 'Data Provider Rows Status field'`

The **pluck** method chooses a specific column to map to. 

Note that the Data Provider rows will be limited by pagination, e.g. the 50 rows being displayed on the current page:
![Screenshot 2022-07-15 at 13 17 28](https://user-images.githubusercontent.com/101575380/179221305-113640fb-c7e3-4e71-830b-ce7f3db0e7b0.png)
